package.go

```diff
@@ -96,7 +97,7 @@ func (m *package_file_cache) process_package_data(data []byte) {
 	// find import section
 	i := bytes.Index(data, []byte{'\n', '$', '$'})
 	if i == -1 {
-		panic("Can't find the import section in the package file")
+		panic(fmt.Sprintf("Can't find the import section in the package file %s", m.name))
```

Comment:
Minor style suggestion, consider using `fmt.Errorf` with `panic` instead of `fmt.Sprintf`. `error` is as nice a type to panic with as `string`, but it communicate intention better, and more easily translates to a future `return fmt.Errorf(...)` if the code is refactored to return error rather than panic.